data_prep/scotland/get_reference_layers.sh

```diff
@@ -66,17 +66,53 @@ function bus_routes {
 }
 
 function population {
-  # From https://www.data.gov.uk/dataset/1102bf85-ed49-440a-b211-da87e8d752eb/scottish-index-of-multiple-deprivation-simd-2020
-  download_to_subdir tmp https://maps.gov.scot/ATOM/shapefiles/SG_SIMD_2020.zip
-  cd tmp
-  unzip SG_SIMD_2020.zip
-  ogr2ogr population.geojson \
-          -t_srs EPSG:4326 \
-          SG_SIMD_2020.shp \
+  if [ ! -f tmp/SG_SIMD_2020.shp ]; then
+      # From https://www.data.gov.uk/dataset/1102bf85-ed49-440a-b211-da87e8d752eb/scottish-index-of-multiple-deprivation-simd-2020
+      download_to_subdir tmp https://maps.gov.scot/ATOM/shapefiles/SG_SIMD_2020.zip
+      unzip -d tmp tmp/SG_SIMD_2020.zip
+  else
+    echo "✅ SG_SIMD_2020.shp already downloaded and extracted."
+  fi
+
+  _car_ownership
+  # We need to join the _car_ownership CSV, so first import both datasets into sqlite
+
```

Comment:
This was a real journey. I spent way too long trying to get an ogr2ogr "oneliner" to work.

Lesson learned: when doing anything non-trivial, consider dumping it into a gpkg (which is a sqlite3 db) and then doing your work in easier to reason about (and validate!) steps.